eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldValue/Handler/Collection.php

```diff
@@ -56,12 +56,30 @@ public function __construct(DatabaseHandler $dbHandler, TransformationProcessor
     public function handle(SelectQuery $query, Criterion $criterion, $column)
     {
         switch ($criterion->operator) {
+            case Criterion\Operator::LIKE:
             case Criterion\Operator::CONTAINS:
+                if ($criterion->operator === Criterion\Operator::LIKE) {
```

Comment:
TBH this doesn't look good, `if` inside `switch-case` repeats case. Judging from context, either entire method should be refactored to get rid of `switch` or some common parts should be extracted and block related to `LIKE` moved to `case LIKE`.

Reply:
Is approach in 99ccb94003da026e649db54a72108803c6b8918a better for you?

Reply:
Looks good now.